onnxruntime/core/session/custom_ops.cc

```diff
@@ -784,14 +788,32 @@ ONNX_NAMESPACE::OpSchema CreateSchema(const std::string& domain, const OrtCustom
       }
     }
 
-    const auto type = op->GetInputType(op, i);
-    if (type == ONNX_TENSOR_ELEMENT_DATA_TYPE_UNDEFINED) {
-      undefined++;
+    std::unordered_set<ONNXTensorElementDataType> all_types;
+    for (auto o : ops) {
+      ORT_ENFORCE(i < o->GetInputTypeCount(o), "Another version of operator '", schema.Name(), "'has less inputs.");
```

Comment:
Please explain 'why' in the error and not just 'what' as it's not immediately obvious how another operator with less inputs renders this invalid.